packages/oscal-react-library/src/components/OSCALAnchorLinkHeader.tsx

```diff
@@ -44,4 +53,6 @@ export default function OSCALAnchorLinkHeader(props) {
       )}
     </Stack>
   );
-}
+};
+
+export default OSCALAnchorLinkHeader;
```

Comment:
I'm wondering if this is the best approach. We're taking a named export (`export const`) and then exporting it as the default export (which is typically used for functions). I would say at the very least remove the `export` in front of `const OSCALAnchorLinkHeader: . . .` above

Reply:
https://github.com/EasyDynamics/oscal-react-library/blob/6c1e060dae048064cb90cea7f587c79f14c2cdca/packages/oscal-react-library/src/components/OSCALMetadata.tsx#L797

This has already been set as a precedence

Reply:
It was removed